clevertap-core/src/main/java/com/clevertap/android/sdk/ProfileValueHandler.kt

```diff
@@ -0,0 +1,221 @@
+package com.clevertap.android.sdk
+
+import com.clevertap.android.sdk.ProfileValueHandler.NumberValueType.DOUBLE_NUMBER
+import com.clevertap.android.sdk.ProfileValueHandler.NumberValueType.FLOAT_NUMBER
+import com.clevertap.android.sdk.ProfileValueHandler.NumberValueType.INT_NUMBER
+import com.clevertap.android.sdk.utils.CTJsonConverter
+import com.clevertap.android.sdk.validation.ValidationResult
+import com.clevertap.android.sdk.validation.ValidationResultFactory
+import com.clevertap.android.sdk.validation.ValidationResultStack
+import com.clevertap.android.sdk.validation.Validator
+import org.json.JSONArray
+
+class ProfileValueHandler(
```

Comment:
does this contain moved code from analytics manager and and java -> kt, is there any change in logic?

Reply:
Logic is the same. Some top-level functions have been added
Other functions have been moved from AnalyticsManager.java